tensorbay/cli/tag.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Implementation of gas tag."""
+import sys
+from typing import Dict
+
+import click
+
+from ..client.gas import DatasetClientType
+from ..client.requests import PagingList
+from ..client.struct import Tag
+from .tbrn import TBRN, TBRNType
+from .utility import get_dataset_client, get_gas
+
+
+def _implement_tag(obj: Dict[str, str], tbrn: str, tag_name: str, is_delete: bool) -> None:
+    gas = get_gas(**obj)
+    if not tbrn:
+        click.echo("Missing argument TBRN", err=True)
+        sys.exit(1)
+
+    info = TBRN(tbrn=tbrn)
+    if info.type != TBRNType.DATASET:
+        click.echo(f'"{tbrn}" is not a dataset', err=True)
+        sys.exit(1)
+
+    dataset_client = get_dataset_client(gas, info)
+
+    if is_delete:
+        pass
+    elif tag_name:
+        pass
+    else:
+        _list_tags(dataset_client)
+
+
+def _list_tags(dataset_client: DatasetClientType) -> None:
+    tags: PagingList[Tag] = dataset_client.list_tags()
```

Comment:
The type `PagingList[Tag]` is redundant, because the return type of `list_tags()` is `PagingList[Tag]`